polaris/dataset/_column.py

```diff
@@ -18,6 +18,12 @@ class Modality(enum.Enum):
     IMAGE = "image"
 
 
+class ContentType(str, enum.Enum):
+    """Used to specify column's IANA content type in a dataset."""
+
+    SMILES = Literal["chemical/x-daylight-smiles"]
```

Comment:
**Comment:**

More of a note for future reference, but I was curious how universal the `chemical/x-daylight-smiles` is.

The `x-daylight-` is throwing me off here. I would expect something more general like `chemical/smiles` (as these [notes](https://www.daylight.com/meetings/summerschool98/course/dave/smiles-relatives.html) suggest as well, see the quote :point_down: ). After a bit of searching it does seem that Daylight invented SMILES, so maybe it's okay? Wikipedia seems to say so ([ref](https://en.wikipedia.org/wiki/Chemical_file_format#The_Chemical_MIME_Project)) as does the [paper](https://pubs.acs.org/doi/10.1021/ci9803233).


> ### The "chemical/smiles" MIME type
> The "chemical" MIME type, championed by H. Rzepa of Imperial College, London, is (hopefully) nearing ratification. Should this happen, SMILES will almost certainly appear as MIME subtype, i.e., the MIME type: chemical/smiles. Until that blessed day, the IETF recommendation is to use something like application/x-smiles or x-chemical/x-smiles.

Reply:
I'm ok with shortening it if we want to. The `chemical` namespace was never ratified officially, even after nearly 30 years, so I don't think it's happening. So we either use the proposed types in https://en.wikipedia.org/wiki/Chemical_file_format, or we use `application/x-[something]`.

For reference, here are all currently official MIME types: https://www.iana.org/assignments/media-types/media-types.xhtml